src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -1884,10 +953,10 @@ bool entitiesQuery
   const std::string&               sortOrderList,
   bool                             includeCreDate,
   bool                             includeModDate,
-  const std::string&               apiVersion
+  const std::string&               apiVersion,
+  StringFilter*                    stringFilterP
```

Comment:
The string filter is now always passed as a Scope
Fixed in many of the latest commits